external/model-preparation-algorithm/mpa_tasks/apis/task.py

```diff
@@ -180,7 +176,7 @@ def _initialize(self, dataset=None, output_model=None):
                 del self._model_cfg._cfg_dict['fp16']
             elif isinstance(self._model_cfg, ConfigDict):
                 del self._model_cfg['fp16']
-        self._precision = self._precision_from_config
+        self._precision = self._model_cfg.get('precision', [ModelPrecision.FP32])
```

Comment:
Does model_cfg always have a precision? It seems that the key 'fp16' is used in the setting (model.py) above, then where do this put the precision to model_cfg?

Reply:
In formal code, _precision only used for what type model has to be exported, orthogonal with training mode(which will be enabled by fp16)

Reply:
Yes, I understand, but I wonder which code allows the key 'precision' to exist in model_cfg.

Reply:
Could you give me some suggestion? I don't think hard-code like [ModelPrecision.FP32] is not good idea, so want to let it get from cfg.

Reply:
I fixed it to get FP32 export option.